hoth_api.yaml

```
@@ -476,28 +476,32 @@ components:
       enum:
         - GB
         - TB
-    RequiredService:
-      description: "Part of DOW, represents a single required service"
+    SelectedServiceOffering:
+      description: "Part of DOW, represents a single Selected Service Offering"
       type: object
       properties:
-        applicable_classification_levels:
+        service_offering:
+          $ref: "#/components/schemas/ServiceOffering"
+        classification_instances:
           type: array
           items:
-            $ref: "#/components/schemas/ClassificationLevel"
+            $ref: "#/components/schemas/ClassificationInstance"
+        other_service_offering:
+          type: string
+    ClassificationInstance:
+      description: "Part of DOW, represents a single classification instance"
+      type: object
+      properties:
         usage_description:
           type: string
-        need_for_entire_to_duration:
-          type: boolean
-        applicable_periods:
+        selected_periods:
           type: array
           items:
             $ref: "#/components/schemas/Period"
-        service_offerings:
-          type: array
-          items:
-            $ref: "#/components/schemas/ServiceOffering"
-        other_service_offering:
-          type: string
+        classification_level:
+          $ref: "#/components/schemas/ClassificationLevel"
+        need_for_entire_to_duration:
```

I was asked to make this small change on schema side for clarity
Might be good to make here as well

```suggestion
        need_for_entire_task_order_duration:
```